### ISSUE TITLE
Feature request: Left hand mode

### ISSUE THREAD

**Issue by flooie** [edemaine/cocreate#194]
A setting to toggle icons to the right hand side for our South pawed brethren would surely be appreciated.

**Comment by flooie** [comment]
I think the only complicated or decision really is where to put the settings or toggle.  Any thoughts.

**Comment by edemaine** [comment]
I feel like a new button just below the touch toggle (another hand icon) makes sense.

**Comment by flooie** [comment]
so what's the icon.  a hand pointing left - and when selected a hand pointing right?
![image](https://user-images.githubusercontent.com/6464529/133006573-fb57ac5b-6c8f-4277-992c-277e799baf1c.png)

**Comment by edemaine** [comment]
Yeah, seems reasonable... or maybe the reverse? (hand points to where the toolbar will go)

I originally hand in mind a left hand holding a pen, which toggles in/out, but I don't see a suitable icon in Font Awesome, so I think I prefer your/this suggestion.
